docs/components/xdczero/design.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebar_position: 1
+---
+
+# design
+
+Achieve unparalleled blockchain interoperability with XDC ZERO, ensuring frictionless data transmission and rigorous validation across a multitude of chains.
+
+## Key Components
```

Comment:
not a big issue but I feel in general the below section uses too much marketing words. 
As a documentation I think it should focus on technical aspect and be straight to the point.